### PR TITLE
Added Affiliations

### PR DESCRIPTION
**Pull request by RLordan**
Hi there, info and affiliation added. Thank you.

### CODE REVIEW

File: content/metadata.yaml [hunk 1]
```diff
@@ -80,3 +80,15 @@ authors:
     coi:
       string: "None"
       lastapproved: !!str 2020-03-22
+  -   
+      github: RLordan
```
Comment (cgreene) [inline content/metadata.yaml]:
```suggestion
    github: RLordan
```

File: content/metadata.yaml [hunk 2]
```diff
@@ -80,3 +80,15 @@ authors:
...
+    name: Ronan Lordan
+    initials: RL
+    orcid: https://orcid.org/0000-0001-9668-3368
```
Comment (cgreene) [inline content/metadata.yaml]:
```suggestion
    orcid: 0000-0001-9668-3368
```

File: content/metadata.yaml [hunk 3]
```diff
@@ -80,3 +80,15 @@ authors:
...
+    twitter: @el_ronan
+    email: ronan.lordan@pennmedicine.upenn.edu
+    contributions: 
+      - Writing - Original Draft
+    affiliations:
+-	Institute for Translational Medicine and Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104-5158, USA
```
Comment (cgreene) [inline content/metadata.yaml]:
```suggestion
      - Institute for Translational Medicine and Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104-5158, USA
```

File: content/metadata.yaml [hunk 4]
```diff
@@ -80,3 +80,15 @@ authors:
...
+    coi: No
```
Comment (cgreene) [inline content/metadata.yaml]:
```suggestion
    coi:
      string: "None"
      lastapproved: !!str 2020-03-25
```

**Comment by agitter** [comment]
I resolved the merge conflict from #51 so we can confirm this builds before merging it.

File: content/metadata.yaml [hunk 5]
```diff
@@ -80,6 +80,20 @@ authors:
     coi:
       string: "None"
       lastapproved: !!str 2020-03-22
+  -   
+    github: RLordan
+    name: Ronan Lordan
+    initials: RL
+    orcid: 0000-0001-9668-3368
+    twitter: @el_ronan
```
Comment (agitter) [inline content/metadata.yaml]:
```suggestion
    twitter: el_ronan
```

Hopefully this will fix the build error

**Pull request merged**